src/main/scala/Bintray.scala

```diff
@@ -12,13 +12,21 @@ object Bintray {
 
   def publishTo(repo: Client#Repo, pkg: Client#Repo#Package, version: String,
     mvnStyle: Boolean = true, isSbtPlugin: Boolean = false, release: Boolean = false): Resolver =
-    if (mvnStyle) RawRepository(
-      BintrayMavenResolver(s"Bintray-Maven-Publish-${repo.subject}-${repo.repo}-${pkg.name}",
-                           s"https://api.bintray.com/maven/${repo.subject}/${repo.repo}/${repo.repo}", pkg, release))
-    else RawRepository(
-      BintrayIvyResolver(s"Bintray-${if (isSbtPlugin) "Sbt" else "Ivy"}-Publish-${repo.subject}-${repo.repo}-${pkg.name}",
-                         pkg.version(version),
-                         sbt.Resolver.ivyStylePatterns.artifactPatterns, release))
+    RawRepository(
```

Comment:
Generally LGTM, but could you highlight the difference between this and #125?
Is it the fact that non-plugin Maven things works as before, or `setM2compatible(true)`?

Reply:
Apologies, I should have explained it better. The difference in this PR is that for sbt plugins with maven style instead of using the _maven upload_ endpoint of Bintray (https://bintray.com/docs/api/#_maven_upload) it uses the _content_ endpoint (https://bintray.com/docs/api/#_upload_content) that is used for ivy style but with the maven layout.
It is a workaround as mentioned in the title as this approach merely side steps the Bintray check for a correct layout through a different api call.
I have no idea if it affects the synchronisation to Central as I don't have an account, but I have successfully published an sbt plugin to Bintray with this change and it gets resolved properly.

Reply:
... and it does keep the previous behaviour for all cases other than `sbtPlugin` with `publishMavenStyle`.

Reply:
Sounds good. Thanks for your contribution!